.github/workflows/go.yml

```diff
@@ -2,26 +2,39 @@ name: Go
 
 on:
   push:
-    branches: [ develop ]
+    branches: [ master ]
```

Comment:
should we change this repo to use 'main' before we merge this?

Reply:
I'd rather not since we have to go through Amelia to make that change, and she's a busy person.